fuzzing/common.bzl

```diff
@@ -56,3 +56,32 @@ Rule for creating a script to run the fuzzing test
     },
     executable = True,
 )
+
+def _fuzzing_corpus_impl(ctx):
+    dir = ctx.actions.declare_directory(ctx.attr.name)
```

Comment:
We should use more explicit variable names to help the reader go through this code more easily. I'll leave suggestions below, feel free to apply the same principle for new code:

`dir` -> `corpus_dir`
`f` -> `input_file`